content/en/serverless/agent_configuration/_index.md

```diff
@@ -0,0 +1,117 @@
+
+| Env Variable                                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`                  | Listen to DogStatsD packets from other containers (required to send custom metrics).                                                                                                                                                                                                                                                                                                                                                                    |
+| `DD_DOGSTATSD_SOCKET`                             | Path to the unix socket to listen to. Must be in a `rw` mounted volume.                                                                                                                                                                                                                                                                                                                                                                                 |
+| `DD_DOGSTATSD_ORIGIN_DETECTION`                   | Enable container detection and tagging for unix socket metrics.                                                                                                                                                                                                                                                                                                                                                                                         |
+| `DD_DOGSTATSD_TAGS`                               | Additional tags to append to all metrics, events, and service checks received by this DogStatsD server, for example: `"env:golden group:retrievers"`.                                                                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| `DD_DOGSTATSD_TAGS`                               | Space-separated list of additional tags to append to all metrics, events, and service checks received by this DogStatsD server. For example: `"env:golden group:retrievers"`.                                                                                                                                                                                                                                                                                                   |
```

does this list have to be surrounded by double-quotes as in the example, or is that a typo?